pytorch_lightning/utilities/model_summary.py

```diff
@@ -0,0 +1,469 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
no further changes to this file except moving it, right?

Reply:
Correct!